src/cascade/executor/dismod_runner.py

```diff
@@ -167,6 +167,20 @@ def run_and_watch(command, single_use_machine, poll_time):
     return loop.run_until_complete(_async_run_and_watch(command, single_use_machine, poll_time))
 
 
+def dismod_report_info(text):
+    """This ensures MATHLOG messages have a function name in the log.
+    Otherwise, they show <lambda> as the function name.
+    """
+    MATHLOG.info(text)
+
+
+def dismod_report_stderr(text):
+    """This ensures MATHLOG messages have a function name in the log.
+    Otherwise, they show <lambda> as the function name.
+    """
+    MATHLOG.warning(text)
```

Comment:
down below, this used to be MATHLOG.error, rather than warning.  Is that intentional?